### PR TITLE
Fix scrolling issues

### PR DESCRIPTION
**Pull request by dting**
Having overflow: auto on the main view didn't give us
    much grainularity in choosing the behavior of each view.

    Refactored the css to allow deciding how the views should
    handle overflow.

### CODE REVIEW

File: client/app/main/about/about.scss [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .about-view {
+  overflow-y: auto;
+  @include cfa-scrollbar(5px);
   .developers,
```
Comment (pdotsani) [inline client/app/main/about/about.scss]: so here you're using `@include` to import a set of styles under the name `cfa-scrollbar`?

File: client/app/main/dashboard/dashboard.scss [hunk 2]
```diff
@@ -24,14 +24,13 @@
       color: #333333
     }
   }
-  .student-list {
+  .student-table {
+    position: relative;
```
Comment (pdotsani) [inline client/app/main/dashboard/dashboard.scss]: Why the change in class name?

File: client/app/main/records/manage/manage.scss [hunk 3]
```diff
@@ -1,169 +1,174 @@
...
-          font-weight: bold;
-        }
+          /* Behave like a "row" */
+          tbody tr td {
+            border: none;
+            border-bottom: 1px solid #eee;
+            position: relative;
+            padding-left: 50%;
+            white-space: normal;
+            text-align: left;
```
Comment (pdotsani) [inline client/app/main/records/manage/manage.scss]: cool... didn't know `white-space` was a property.

File: client/app/main/school/school.reports.controller.js [hunk 4]
```diff
@@ -2,7 +2,8 @@
 
 var app = angular.module('app');
 
-function SchoolReportsCtrl($scope, $timeout, Auth, uiGridGroupingConstants, Student,
+function SchoolReportsCtrl($scope, $timeout, Auth, uiGridGroupingConstants,
+  Student,
   AbsenceRecord, toastr) {
   $scope.atRiskGridOptions = {
```
Comment (pdotsani) [inline client/app/main/school/school.reports.controller.js]: Indentation looks a little messy here.

**Comment by pdotsani** [comment]
lgtm. Just a few notes.

For future reference, is there a max character width I should be aware of when it comes to formatting?

File: client/app/main/about/about.scss (hunk 1 above)
Comment (dting) [inline client/app/main/about/about.scss]: @include is for using a mixin.

http://krasimirtsonev.com/blog/article/SASS-mixins-extends-and-placeholders-differences-use-cases
http://roytomeij.com/blog/2013/should-you-use-a-sass-mixin-or-extend.html
https://tech.bellycard.com/blog/sass-mixins-vs-extends-the-data/

File: client/app/main/dashboard/dashboard.scss (hunk 2 above)
Comment (dting) [inline client/app/main/dashboard/dashboard.scss]: More descriptive of the div.

**Comment by dting** [comment]
The javascript code is formatted to 80 characters wide. The other file types are less strict.
